controllers/idler/idler_controller.go

```diff
@@ -198,6 +181,58 @@ func (r *Reconciler) ensureIdling(ctx context.Context, idler *toolchainv1alpha1.
 	return r.updateStatusPods(ctx, idler, newStatusPods)
 }
 
+// Check if the pod belongs to a controller (Deployment, DeploymentConfig, etc) and scale it down to zero.
+// if it is a standalone pod, delete it.
+// Send notification if the deleted pod was managed by a controller, was a standalone pod that was not completed or was crashlooping
+func deletePodsAndCreateNotification(ctx context.Context, podLogger logr.Logger, pod corev1.Pod, r *Reconciler, idler *toolchainv1alpha1.Idler) error {
+	lctx := log.IntoContext(ctx, podLogger)
+	logger := log.FromContext(ctx)
+	var podreason string
```

Comment:
```suggestion
	var podReason string
```

Reply:
done in [9164efe](https://github.com/codeready-toolchain/member-operator/pull/630/commits/9164efed1c4569ec9780def2f3ef716fc2da1abe)